packages/service-library/src/servicelib/redis.py

```diff
@@ -178,9 +184,11 @@ class RedisClientSDKHealthChecked(RedisClientSDK):
     def __init__(
         self,
         redis_dsn: str,
-        health_check_interval: datetime.timedelta = datetime.timedelta(seconds=5),
+        *,
+        decode_responses: bool = _DEFAULT_DECODE_RESPONSES,
+        health_check_interval: datetime.timedelta = _DEFAULT_HEALTH_CHECK_INTERVAL,
     ) -> None:
-        super().__init__(redis_dsn)
+        super().__init__(redis_dsn, decode_responses)
         self.health_check_interval: datetime.timedelta = health_check_interval
         self._health_check_task: Task | None = None
         self._is_healthy: bool = True
```

Comment:
it is a dataclass, why do you need to put this init code???

Reply:
Good observation that was not required.